Sources/PrimerSDK/Classes/Core/Primer/PrimerDelegate.swift

```diff
@@ -29,15 +29,17 @@ public protocol PrimerDelegate {
     
     func clientTokenCallback(_ completion: @escaping (_ token: String?, _ error: Error?) -> Void)
     
-    @objc optional func tokenAddedToVault(_ token: PaymentMethodToken)
+    @objc optional func tokenAddedToVault(_ token: PaymentMethod)
     
     /// This function will be called when the user tries to make a payment. You should make the pay API call to your backend, and
     /// pass an error or nil on completion. This way the SDK will show the error passed on the modal view controller.
     ///
     /// - Parameters:
-    ///   - paymentMethodToken: The PaymentMethodToken object containing the token's information.
+    ///   - paymentMethodToken: The PaymentInstrument object containing the token's information.
     ///   - completion: Call with error or nil when the pay API call returns a result.
-    @objc optional func onTokenizeSuccess(_ paymentMethodToken: PaymentMethodToken, _ completion:  @escaping (Error?) -> Void)
+//    @available(*, deprecated, renamed: "onTokenizeSuccess")
+//    @objc optional func onTokenizeSuccess(_ paymentMethodToken: PaymentMethod, _ completion:  @escaping (Error?) -> Void)
+    @objc optional func onTokenizeSuccess(_ paymentMethod: PaymentMethod, _ completion:  @escaping (Error?) -> Void)
```

Comment:
Did we align on the PaymentInstrument and PaymentMethod naming convention? Just confirming since tokenize success refers technically to a payment instrument of type payment method.